package.json

```
@@ -22,10 +22,12 @@
   },
   "main": "lib/index.js",
   "dependencies": {
+    "@types/node-fetch": "^2.5.7",
```

`devDependencies`の方が良いかも？
```suggestion
    "@types/node-fetch": "^2.5.7",
```

---

たし🦀